db/schema.rb

```diff
@@ -1142,6 +1143,8 @@
   add_foreign_key "cohorts", "tracks"
   add_foreign_key "community_stories", "users", column: "interviewee_id"
   add_foreign_key "community_stories", "users", column: "interviewer_id"
+  add_foreign_key "community_videos", "exercises"
+  add_foreign_key "community_videos", "tracks"
```

Comment:
These lines were automatically added and it seems like they should indeed have been there.